### ISSUE TITLE
Remove "rights lock" feature

### ISSUE THREAD

**Issue by trustmaster** [Cotonti/Cotonti#1230]
The "Rights lock" feature has been there since Seditio (or even LDU?) but to us it's been doing more harm than help: permissions get locked here and there and you need to unlock them manually via phpMyAdmin.

Maybe it's time just to get rid of them and be able to set any permissions everywhere.

**Comment by Alex300** [comment]
I don't agree with it. We should keep this feature. It protect important rights setting from an inexperienced site administrator.
